device/globals/dialog_instance.gd

```diff
@@ -79,17 +77,17 @@ func init(p_params, p_context, p_intro, p_outro):
 	var sep = text.find(":\"")
 	var text_id = null
 	if sep > 0:
-		var tid = text.substr(0, sep)
+		text_id = text.substr(0, sep)
 		text = text.substr(sep + 2, text.length() - (sep + 2))
 
-		var ptext = TranslationServer.translate(tid)
-		if ptext != tid:
+		var ptext = TranslationServer.translate(text_id)
+		if ptext != text_id:
 			text = ptext
-		elif force_ids:
-			text = tid + " (" + text + ")"
-		text_id = tid
+		else:
+			text = "(NOT TRANSLATED)\n\n" + text
```

Comment:
Shouldn't this also use `vm.report_errors()´, similar to if text_id is not provided on line 90? For the cowboy devs, I think the dialogue should appear similarly in the two cases, eg. `"(NO TRANSLATION)\n\n" + text` and `"(NO TEXT_ID)\n\n" + text` OR `"(NO TRANSLATION) " + text` and `"(NO TEXT_ID)" + text`

Reply:
Ok, I agree with your reasoning. 👍